doc/src/manual/modules.md

```diff
@@ -260,7 +260,15 @@ import BenchmarkTools as BT
 
 `as` works with `using` only when a single identifier is brought into scope.
 For example `using CSV: read as rd` works, but `using CSV as C` does not, since it operates
-on all of the exported names in `CSV`.
+on all of the exported names in `CSV`. Packages can also be renamed to `_`, for example
+
+```julia
+import ForwardDiff as _
+```
+
```

Comment:
Perhaps also give the variant with `using` instead of `import`?

````suggestion

Or, with `using`:

```julia
using ForwardDiff: ForwardDiff as _
```

````

Reply:
on nightly it gives a different warning:
```julia
julia> "ForwardDiff" in string.(collect(values(Base.loaded_modules)))
false

julia> using ForwardDiff: _
WARNING: Imported binding ForwardDiff._ was undeclared at import time during import to Main.

julia> "ForwardDiff" in string.(collect(values(Base.loaded_modules)))
true
```
but again still works